<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>596</width>
    <height>201</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>596</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ust. połączenia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButtonExit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Zamknij</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>337</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEditPort_1">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Wprowadź numer portu, po którym aplikacja będzie łączyć się z drukarką.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Przykład:&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;6666&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>Port: 6666</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButtonRefresh_1">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Odśwież</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButtonConnect_1">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Połącz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="labelSpeed_1">
           <property name="text">
            <string>Prędkość</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxConnectionSpeed_1">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string notr="true">115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelFlowControl_1">
           <property name="text">
            <string>Sterowanie
przepływem</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxFlowControl_1">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Brak</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Xon/Xoff</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RTS/CTS</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelParity_1">
           <property name="text">
            <string>Parzystość</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxParity_1">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Brak</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Parzyste</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nieparzyste</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButtonDisconnect_1">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Rozłącz</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButtonCom_1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>COM</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonTcp_1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TCP</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxPortName_1">
         <property name="minimumSize">
          <size>
           <width>421</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditIp_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wprowadź adres IP drukarki.&lt;/p&gt;&lt;p&gt;Format:&lt;/p&gt;&lt;p&gt;10.2.3.123&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>IP: 10.2.3.123</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButtonMemoryTester">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Tester
pamięci</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelConnectionInfo">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stan połączenia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelConnectionColor">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelConnectionStatus">
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Brak połączenia</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="labelVersion">
      <property name="text">
       <string>Wer. 0.9.3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButtonDataMonitor">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Monitor
portu</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButtonConfiguration">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Konfiguracja</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
